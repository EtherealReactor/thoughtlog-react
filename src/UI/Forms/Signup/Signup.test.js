import React from 'react';
import { shallow, mount } from 'enzyme';
import shallowToJson from 'enzyme-to-json';
import SignupForm from './Signup';

describe("FormComponent === Signup", () => {
  it("must render SignupForm", () => {
    const wrapper = shallow(<SignupForm />)
    expect(wrapper).toHaveLength(1);
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});