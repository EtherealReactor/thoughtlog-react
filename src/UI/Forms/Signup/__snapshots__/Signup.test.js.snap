// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormComponent === Signup must render SignupForm 1`] = `
<Connect(Form(SignupForm))
  destroyOnUnmount={true}
  enableReinitialize={false}
  forceUnregisterOnUnmount={false}
  form="SignupForm"
  getFormState={[Function]}
  keepDirtyOnReinitialize={false}
  persistentSubmitErrors={false}
  pure={true}
  shouldAsyncValidate={[Function]}
  shouldError={[Function]}
  shouldValidate={[Function]}
  shouldWarn={[Function]}
  touchOnBlur={true}
  touchOnChange={false}
  updateUnregisteredFields={false}
  validate={[Function]}
/>
`;
